### PR TITLE
return promise to fix v6 CLI

### PR DESCRIPTION
**Pull request by dickeyxxx**

### CODE REVIEW

**Comment by ransombriggs** [comment]
@dickeyxxx `if (bastions != null)` does not return a promise

**Comment by dickeyxxx** [comment]
fixed the code and test to work with bastions but I don't have an app handy to actually test it on. @cyberdelia would you mind running this branch on a bastion redis app to make sure it still works?

**Comment by cyberdelia** [comment]
Looks like it works.

**Comment by dickeyxxx** [comment]
thanks @cyberdelia!

File: commands/cli.js [hunk 1]
```diff
@@ -88,31 +88,17 @@ function redisCLI (uri, client) {
...
-function maybeTunnel (redis, config) {
-  let bastions = match(config, /_BASTIONS/)
-  let hobby = redis.plan.indexOf('hobby') === 0
-  let uri = url.parse(redis.resource_url)
-
-  if (bastions != null) {
+function bastionConnect ({uri, bastions, config}) {
+  return new Promise((resolve, reject) => {
     let tunnel = new Client()
     tunnel.on('ready', function () {
```
Comment (ransombriggs) [inline commands/cli.js]: @dickeyxxx you need to handle an error here since if we never get a `ready` event I think this will hang indefinitely

Comment (ransombriggs) [inline commands/cli.js]: @dickeyxxx not sure of the API, but I assume you should also reject here?
```
        if (err) {
          cli.error(err)
        }
```

**Pull request merged**